src/Gui/TextEdit.cpp

```diff
@@ -495,6 +495,30 @@ PythonTextEditor::PythonTextEditor(QWidget *parent)
 
 PythonTextEditor::~PythonTextEditor() = default;
 
+void PythonTextEditor::onComment(const QString& ch)

```

Comment:
I don't know about keeping the `onComment` name here: in one place it's used for a comment, but in another for whitespace. The use of "ch" as the parameter name also feels misleading to me, I had to go back and double-check that it did indeed work with multiple characters.

Reply:
@benj5378 what do you think about changing the name?